webdriver/tests/websockets/connect.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import asyncio
+import websockets
+
+@pytest.mark.asyncio
+@pytest.mark.capabilities({"webSocketUrl": True})
+async def test_websocket_url_connect(session):
+    websocket_url = session.capabilities["webSocketUrl"]
+    async with websockets.connect(websocket_url) as websocket:
+        await websocket.send("Hello world!")
```

Comment:
The idea here is that this can end up returning an error on the socket, but we don't care?

Anyway it seems reasonable. I don't think the commit message should mention Chrome though; this is surely for any bidi implementation.

Reply:
Yes, that's the idea.
I will replace ChromeDriver with WebDriver.